alacritty/src/logging.rs

```diff
@@ -149,6 +149,16 @@ fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
     message
 }
 
+fn is_allowed_target(level: Level, target: &str) -> bool {
```

Comment:
```suggestion
/// Check if log messages from a crate should be logged.
fn is_allowed_target(level: Level, target: &str) -> bool {
```

Wording might be a bit awkward, but I think a little comment would be nice in case it's not immediately obvious what "allowed target" means in this context.

Reply:
Added comment to latest commit.